nebula/ui/components/MZBottomSheet.qml

```diff
@@ -17,7 +17,7 @@ import Mozilla.Shared 1.0
       contentItem: ColumnLayout {
           Rectangle {
               anchors.left: parent.left
-              anchors. right: parent.right
+              anchors.right: parent.right
```

Comment:
This is just documentation, so doesn't change anything. (If you expand this snippet you'll see it's a comment.) I noticed this typo, and fixed it.